sormas-backend/src/main/java/de/symeda/sormas/backend/audit/AuditLoggerInterceptor.java

```diff
@@ -32,14 +38,23 @@
 
 public class AuditLoggerInterceptor {
 
-	private final Logger logger = LoggerFactory.getLogger(getClass());
 	@EJB
 	AuditLoggerEjb.AuditLoggerEjbLocal auditLogger;
 
+	private static final Reflections reflections;
+	static {
+
+		ConfigurationBuilder configurationBuilder = new ConfigurationBuilder();
+		configurationBuilder.forPackages("de.symeda.sormas.backend").addScanners(SubTypes).setParallel(false);
+		reflections = new Reflections(configurationBuilder);
+	}
+
+	private static final Set<Class<?>> adoServiceClasses = new HashSet<>(reflections.get(SubTypes.of(BaseAdoService.class).asClass()));
```

Comment:
Please do this within the static initialize of the class

Reply:
done